### PR TITLE
Dedicated world model for the valve demo (BOP) with more ambient light

### PR DESCRIPTION
**Pull request by bsb808**
With bdd68841b6f2a3135a4b68150db7ce05eb7349b0 Youssef added more ambient light to support the BOP example: [BOP-Panel-Manipulation-Mission](BOP-Panel-Manipulation-Mission).   The additional light was added to the default world, so for other examples the lighting is overexposed.

This PR makes a dedicated world for the BOP example.

# To Test

## BOP World

```
roslaunch uuv_dave valve_manipulation_demo.launch
```
Gazebo_084

## Nominal World

```
roslaunch uuv_dave uuv_dave.launch 
```

Gazebo_083

### CODE REVIEW

File: uuv_dave/launch/valve_manipulation_demo.launch [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <launch>
     <arg name="gui" default="true"/>
     <arg name="paused" default="false"/>
-    <arg name="world_name" default="$(find uuv_dave)/worlds/uuv_dave_ocean_waves.world"/>
+    <arg name="world_name" default="$(find uuv_dave)/worlds/valve_manipulation_demo.world"/>
     <arg name="namespace" default="rexrov"/>
```
[the remark this answers is not in the document]
Comment (bsb808) [reply uuv_dave/launch/valve_manipulation_demo.launch:6]: Darn - I hate it when I do that.  Should be added.

**Comment by M1chaelM** [comment]
@bsb808 This works fine now except in the dedicated BOP world the camera starts in the same place as the normal world, so I am looking at nothing until you pan out. Do we need to adjust a start position setting?

**Comment by bsb808** [comment]
@M1chaelM I moved the camera.  For these kinds of tweaks, consider putting in your own commit to save some time.

**Comment by M1chaelM** [comment]
@bsb808 Sure. I haven't messed with the camera before; one of my many blind spots. If I know how to fix whatever is wrong I will do that. Anyway, this looks good now.

**Review by M1chaelM — APPROVED** [review]
Works as expected.

**Comment by bsb808** [comment]
> @bsb808 Sure. I haven't messed with the camera before; one of my many blind spots. If I know how to fix whatever is wrong I will do that. Anyway, this looks good now.

@M1chaelM The initial pose is specified in the world file.  I use the GUI to get the view I want and then copy the values to the world file.

**Pull request merged**